docs/faq/proof-of-stake/general.md

```diff
@@ -0,0 +1,178 @@
+# **<i class="fa fa-ticket"></i> PoS General FAQ**
+
+---
+
+#### **1. What is Decred's Proof-of-Stake system?**
+
+The Proof-of-Stake (PoS) system allows holders of Decred to have a say in the governance of the currency. It aims to take full control away from the miners and allow the users to have a voice.
```

Comment:
Could we phrase the 2nd sentence a little differently.  Usually we've said that the miners and the devs have total control in bitcoin rather than the miners having total control (and it is a little vague when phrased that way).  Maybe 'It aims to give the users a voice along side the miners.' or something like that.